test/WrapperTest.sol

```diff
@@ -185,4 +178,6 @@ contract MorphoTokenEthereumMigrationTest is BaseTest {
 
 interface RolesAuthority {
```

Comment:
Probably consider moving this at the top of the file, I was looking for it add the beginning.